app/views/posts/show.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Display the specific post for the user</h1>
\ No newline at end of file
+<div class="post-container">
+  <h2>Post #<%= @post.id %> by <%= @post.author.name %></h2>
+  <div class='userPost'>
+    <p><%= @post.text %></p>
+  </div>
+  <p class="coli">Comments: <%= @post.comments.count %>, Likes: <%= @post.likes.count %></p>
+</div>
+
+<h2>Comments</h2>
+<div class="comment-list">
```

Comment:
- [ ] Just as I have suggested above, You need to make sure your design follows the given wireframe, Your current design doesn't follow the layout completely, I'll suggest you update this correctly. :+1: :100: 

- - Your current Design:
![Selection_042](https://github.com/faisalakb/Rails-Blog-App/assets/107632780/e4bfe5fd-2c02-4008-a1b2-b01462299905)


- - Wireframe:
![Screenshot from 2023-11-24 09-30-19](https://github.com/faisalakb/Rails-Blog-App/assets/107632780/b1ada432-f850-4353-9d35-6670fa0a10ce)